Extensions/xkit_patches.js

```diff
@@ -1850,20 +1866,14 @@ XKit.tools.dump_config = function(){
 		// New Post Listener for Posts_v2
 		XKit.post_listener.check = function(no_timeout) {
 			var post_count = -1;
-			if (typeof XKit.page.peepr != "undefined" && XKit.page.peepr === true) {
-				post_count = $(".post").length;
-			} else {
-				if ($("#posts").length === 0) {
-					return;
-				}
-				post_count = $("#posts .post").length;
-			}
+
+			post_count = $(".posts .post").length;
```

Comment:
see https://github.com/new-xkit/XKit/pull/521/files#diff-f3c602f5caa159014c2f7557700de10cL1859 in the previous changeset